pkg/api/mlflow/dao/convertors/log_test.go

```diff
@@ -74,6 +74,8 @@ func TestConvertLogBatchRequestToDBModel_Ok(t *testing.T) {
 					Timestamp: 1234567890,
 					RunID:     "run_id",
 					Step:      1,
+					ContextID: models.DefaultContextID,
```

Comment:
I'm not seeing the whole file in this diff, but seeing this repeated four times makes my spider sense tingle. Any way to DRY it up?

Reply:
well, these are "expected results" which are compared to the function output -- every field has to be provided. Using the `models.DefaultContext` var and const do help a little.